apps/client/project.json

```diff
@@ -17,7 +17,7 @@
         "polyfills": "apps/client/src/polyfills.ts",
         "tsConfig": "apps/client/tsconfig.app.json",
         "assets": ["apps/client/src/favicon.ico"],
-        "styles": ["apps/client/src/styles.scss"],
+        "styles": ["apps/client/src/styles.css"],
```

Comment:
why are we switching to css?

Reply:
We are using tailwind `css`, so we don't need external styles. `scss` was initialized from the beginning of the project. Switching back to `css` will avoid the use of `sass-loader` as well.